.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+[//]: #  (Link to the issue corresponding to this chunk of work)
+:tickets: Fixes __issue__
+:scroll: Design Doc: __link if applicable__
```

Comment:
This is a public repo, so I like to precede these two bullets with something like `_Eppo Internal:_`

Reply:
done!